dashboard/app/views/layouts/_header.html.haml

```diff
@@ -25,6 +25,18 @@
               = render partial: 'shared/header_progress'
             - elsif @edit_blocks
               .header_level{style:'font-size: 250%; line-height: 125%;'} Editing #{@edit_blocks.humanize}
-          - if params[:controller] !~ /^sessions/
-            .header_user
-              = render partial: 'shared/user_header'
+          .header_right
+            %span.stage_links{style: 'display: none;'}
```

Comment:
Hm, I think in Sarah's mock, it's ambiguous where the freeplay_links should show up. I see them on slide 2 which is the homepage (which is why I was thinking that the mocks were showing them all the time) but do not see them on slide 7 which is also the homepage. Maybe ask her?